src/stub.c

```diff
@@ -366,24 +374,62 @@ int CALLBACK _tWinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPTSTR lpCm
    return 0;
 }
 
+static PIMAGE_NT_HEADERS retrieveNTHeader(LPVOID ptr) {
```

Comment:
Brief description of what the `NTheader` is and how it's located would be great.

Reply:
@jakedouglas will do